src/main/resources/config.yml

```diff
@@ -372,10 +372,10 @@ message-could-not-unload-blacklist: "&7Nothing to unload: All items in your inve
 
 ##### Russian
 #blacklist-title: "----- &cЧерный список&r -----"
-#message-could-not-unload: "&7Нечего выгружать: Нет сундуков для оставшихся предметов."
-#message-radius-too-high: "&cОшибка:&7 Радиус не может быть больше %d блоков."
-#message-no-chests-nearby: "&7Нечего выгружать: Нет сундуков поблизости. Измените радиус или подойдите ближе к сундукам."
-#message-error-not-a-number: "&cОшибка:&7 '%s' недопустимое число."
+#message-could-not-unload: "&7Нечего выгружать: нет сундуков для оставшихся предметов."
+#message-radius-too-high: "&cОшибка:&7 радиус не может быть больше %d блоков."
+#message-no-chests-nearby: "&7Нечего выгружать: нет сундуков поблизости. Измените радиус или подойдите ближе к сундукам."
```

Comment:
Shouldn't letters start with uppercase after colons?

Reply:
Grammatically they shouldn't. It's not a beginning of a new sentence.